libs/wing-fixture/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "wing-fixture",
+  "version": "0.0.0",
+  "description": "A fake library just for testing stuff out",
+  "author": {
+    "name": "Wing Cloud",
+    "email": "ping@monada.co",
+    "url": "https://wing.cloud",
+    "organization": true
+  },
+  "files": [
+    "**/*.js",
+    "**/*.w",
+    "!/target/**"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">=v18.0.0"
+  },
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/winglang/wing.git"
+  },
+  "scripts": {
+    "package": "npm pack"
+  },
+  "wing": {
+    "entrypoint": "lib.w"
+  },
```

Comment:
@eladb besides simplifying the implementation, my initial feeling is that having a single file represent the "entrypoint" of a module could be a good practice since adding or removing exports is in a single place. (versus, if we inferred that all files at the root of your project are public modules, then you would have to move a Wing file into a folder in order to hide it / make it private). This sort of aligns with the some parts of the language's philosophy around "private by default".

WDYT / how are you thinking about the DX tradeoffs? (I'm not totally sold on either direction personally)

Reply:
Once we have access modifiers, only types that are marked `pub` will be part of the library's public API. The fact that something is under a subdirectory does not imply that it's private.